src/citrine/resources/response.py

```diff
@@ -22,7 +22,7 @@ def _get_body_string(self):
             return "No body available"
 
     def __repr__(self):
-        return 'Response({!r}), {!r})'.format(self._get_status_string(), self._get_body_string())
+        return 'Response({!r}, {!r})'.format(self._get_status_string(), self._get_body_string())
```

Comment:
As long as you're editing, why not migrate to an f-string?

Reply:
Good call! This was in my `git stash` from a ways back, so I didn't even think to do that.